pkg/templating/template_description.go

```diff
@@ -33,8 +36,98 @@ func TemplateStringRaw(inputTemplate string, issue *v1.Issue) (*string, error) {
 	return &res, nil
 }
 
+type enrichedIssue struct {
+	*v1.EnrichedIssue
+	ToolName       string
+	ScanStartTime  string
+	ScanID         string
+	ConfidenceText string
+	SeverityText   string
+	Count          int
+	FirstFound     string
+}
+type enrichedIssueOption func(*enrichedIssue) error
+
+func EnrichedIssueWithToolName(toolname string) enrichedIssueOption {
+	return func(ei *enrichedIssue) error {
+		if toolname == "" {
+			return errors.New("invalid toolname ''")
+		}
+		ei.ToolName = toolname
+		return nil
+	}
+}
+
+func EnrichedIssueWithScanStartTime(startTime *time.Time) enrichedIssueOption {
```

Comment:
Perhaps this should be a pointer? 

It would be not super ergonomic to use from an API POV. Imagine having to pass a reference to a `time.Date`

I'd keep a value and check that it's not a 0 value :)